package.json

```diff
@@ -117,7 +117,7 @@
   "devDependencies": {
     "@apollo/server": "^4.11.0",
     "@eslint/eslintrc": "^3.1.0",
-    "@eslint/js": "^9.11.1",
+    "@eslint/js": "^8.57.1",
```

Comment:
This version of is actually not compatible with the major version of ESLint we use (v8). Once we upgrade to v9, we can upgrade this as well